raiden/transfer/mediated_transfer/mediator.py

```diff
@@ -190,6 +190,24 @@ def filter_used_routes(
     return list(channelid_to_route.values())
 
 
+def payer_transfer_expired(
+        payer_channel: NettingChannelState,
+        pair: MediationPairState,
+        block_number: typing.BlockNumber,
+) -> bool:
+    payer_lock_expiration_threshold = typing.BlockNumber(
+        pair.payer_transfer.lock.expiration +
+        DEFAULT_NUMBER_OF_BLOCK_CONFIRMATIONS * 2,
+    )
+    has_payer_lock_expired, _ = channel.is_lock_expired(
+        end_state=payer_channel.our_state,
+        lock=pair.payer_transfer.lock,
+        block_number=block_number,
+        lock_expiration_threshold=payer_lock_expiration_threshold,
+    )
+    return has_payer_lock_expired
```

Comment:
Note: The check here from the previous version of the function does not check for the `payer_state!='expired'`